src/System.Private.Xml/src/System/Xml/Serialization/Xmlcustomformatter.cs

```diff
@@ -77,7 +77,7 @@ internal static string FromDefaultValue(object value, string formatter)
                     return FromXmlNmTokens((string)value);
                 }
             }
-            throw new Exception(SR.Format(SR.XmlUnsupportedDefaultType, type.FullName));
+            throw new NotSupportedException(SR.Format(SR.XmlUnsupportedDefaultType, type.FullName));
```

Comment:
I'd vote for XmlException instead of this.

Reply:
actually this seems to match style in this file, so I'm ok with this

Reply:
found InvalidOperationException only in src\System.Private.Xml\src\System\Xml\Dom\XmlNotation.cs
 ```csharp
  // Throws an InvalidOperationException since Notation can not be cloned.
        public override XmlNode CloneNode(bool deep)
        {
            throw new InvalidOperationException(SR.Xdom_Node_Cloning);
        }
 ```